SystemInformer/findobj.c

```diff
@@ -734,6 +734,7 @@ VOID PhpFindObjectAddResultEntries(
         objectNode->BestObjectName = searchResult->BestObjectName;
         objectNode->HandleInfo = searchResult->Info;
         PhPrintPointer(objectNode->HandleString, searchResult->Handle);
+        objectNode->HandleInfo.Reserved = ULONG_MAX;
```

Comment:
Is this necessary to keep?

Reply:
Yes. hndllist.c and hndlprp.c will not show RefCnt in this case (RefCnt can be zero and it is valid value, but not ULONG_MAX)
```
            case PHHNTLC_POINTERCNT:
                if (handleItem->RefCnt != ULONG_MAX)
                    PhInitializeStringRefLongHint(&getCellText->Text, handleItem->PointerCountString);
```

Reply:
A handle itself doesn't have a pointer count though? I guess I'm not clear on why this still here since we won't show anything?

Reply:
If driver decode pointer count for us we can show handle pointer count. I removed driver changes, so now it will show nothing.

Reply:
Understood. The "reference count" in the handle isn't really a reference count, it's very poorly named. It would be better described as a "use count". Even then it's an inverted value... Please just remove the dead code for now until we determine how to handle that.